qiskit_experiments/framework/composite/parallel_experiment.py

```diff
@@ -49,6 +49,7 @@ def __init__(
         backend: Optional[Backend] = None,
         flatten_results: bool = None,
         analysis: Optional[CompositeAnalysis] = None,
+        experiment_type: Optional[str] = None,
```

Comment:
Remember to also pass it to the parent's constructor

Reply:
ahhh I didn't catch that before 🤦‍♂️